gateway/src/main.rs

```diff
@@ -46,11 +46,14 @@ async fn main() {
         .route("/feedback", post(endpoints::feedback::feedback_handler))
         .route("/status", get(endpoints::status::status_handler))
         .route("/health", get(endpoints::status::health_handler))
-        .route(
+        .with_state(app_state);
+
+    if let Some(metrics_handle) = metrics_handle {
+        router = router.route(
             "/metrics",
             get(move || std::future::ready(metrics_handle.render())),
         )
-        .with_state(app_state);
+    }
```

Comment:
Metrics shouldn't fail because of clickhouse, no need?

Reply:
I was trying to be as conservative as possible about keeping the gateway up but I guess it's OK

Reply:
After investigating, this only fails if you've already exported the metrics. OK to keep an expect